container/helm/charts/webui/templates/_helpers.tpl

```diff
@@ -0,0 +1,51 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "webui.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
```

Comment:
I can only guess the appropriate articles (the/a):
```suggestion
If the release name contains a chart name it will be used as a full name.
```

Reply:
Nice catch, thanks. I did not study this file much (it is from helm template).